app/Models/Forum/Topic.php

```diff
@@ -567,6 +567,15 @@ public function isLocked()
         return $this->topic_status !== static::STATUS_UNLOCKED;
     }
 
+    public function isActive()
+    {
+        return $this
+            ->topic_last_post_time
+            ->copy()
```

Comment:
I should point out that this will die if `topic_last_post_time` returns `null` if for whatever reason the db value is 0

Reply:
probably not worth handling it at the moment, since the parts that can actually make it `null` have other issues which actually prevent it from happening